docs/how-to/import-prisma-schema.md

```diff
@@ -0,0 +1,120 @@
+---
+id: import-prisma-schema
+title: Import Prisma Schema
+sidebar_label: Import Prisma Schema
+slug: /how-to/import-prisma-schema
+pagination_next: getting-started/how-prisma-schema-is-converted-into-entities
+---
+
+# Import Prisma Schema
+
+Amplication's **Import Prisma Schema** feature lets you use your _existing_ databases for a head start on development.
+
+This way you don't have to manually create entities, fields, and relationships from scratch saving you a lot of time.
```

Comment:
We should also talk about a modernization project where you have an existing DB and want to create a new application or API on top of it